lib/house.rb

```diff
@@ -0,0 +1,42 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @above_market_average = false
+    @sum = 0
+    @area_room = []
+    @detail_hsh = {}
```

Comment:
You don't need any of these instance variables either-- everything you want to do can be accomplished with methods